crates/bevy_tasks/src/task_pool.rs

```diff
@@ -539,6 +546,34 @@ impl TaskPool {
         Task::new(self.executor.spawn(future))
     }
 
+    /// Runs the provided closure on a thread where blocking is acceptable.
+    ///
+    /// In general, issuing a blocking call or performing a lot of compute in a
+    /// future without yielding is not okay, as it may prevent the task pool
+    /// from driving other futures forward. This function runs the provided
+    /// closure on a thread dedicated to blocking operations.
+    ///
+    /// This call will spawn more blocking threads when they are requested
+    /// through /// this function until the upper limit configured. This
+    /// limit is very large by default, because `spawn_blocking` is often
```

Comment:
```suggestion
    /// limit is very large by default, because `spawn_blocking` is often
```
I think it worth stating the exact default number and also documenting how to increase this number (using `TaskPoolBuilder` or `BLOCKING_MAX_THREADS` env var

Reply:
Done!